src/main/resources/project/server/checkServerGroupStatus.pl

```diff
@@ -18,17 +18,28 @@ =head1 COPYRIGHT
 my $PLUGIN_NAME = '@PLUGIN_NAME@';
 my $PLUGIN_KEY = '@PLUGIN_KEY@';
 
+my $PARAM_CRITERIA_STARTED = 'STARTED';
+my $PARAM_CRITERIA_STOPPED_OR_DISABLED = 'STOPPED';
+
+my %PARAM_CRITERIA_LABELS = (
+    $PARAM_CRITERIA_STARTED             => "STARTED",
+    $PARAM_CRITERIA_STOPPED_OR_DISABLED => "STOPPED",
+);
+
+my $STATUS_STOPPED = 'STOPPED'; # stopped status for servers with auto-start true
+my $STATUS_DISABLED = 'DISABLED'; # stopped status for servers with auto-start false
+my $STATUS_STARTED = 'STARTED'; # started status for servers
```

Comment:
For now it is not required, but if we have a lot of constants, maybe, we need to have special lib with constants?